### PR TITLE
Remove get_jwt_token_full_run() test

### PR DESCRIPTION
**Pull request by matthiasbeyer**
## Proposed changes

This test constantly blocks CI because it fails unreproducably. This patch removes it, so we can get rid of these sporadic failures.

Closes #853 

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

IMHO we should remove all tests that access network.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
I hope that this is the only "flaky" testcase.

**Review by didier-wenzek — COMMENTED** [review]
It makes sense. However, Pradeep is testing a fix. I agreed with him: if this is not fixed today, I will we be happy to approve this PR.

File: crates/core/tedge_mapper/src/c8y/tests.rs [hunk 1]
```diff
@@ -332,38 +332,6 @@ async fn mapper_publishes_software_update_request_with_wrong_action() {
     .await;
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```
[the remark this answers is not in the document]
Comment (matthiasbeyer) [reply crates/core/tedge_mapper/src/c8y/tests.rs:335]: Very good catch!

**Comment by matthiasbeyer** [comment]
@didier-wenzek Do you mean that we should merge this in favour of #853 and remove the test? I'm not sure whether this was meant by your previous comment :thinking: 

If yes, feel free to push the button :tada:

**Pull request merged**